README.rst

```diff
@@ -23,20 +23,31 @@ Building is as simple as
 ::
 
         $ git clone https://github.com/FreeRCT/FreeRCT.git
-        $ cd FreeRCT # Go into the downloaded source directory.
-        $ cmake .    # Checks libraries are where they're supposed to be and replaces some strings
-        $ make       # Let make do the heavy work.
+        $ cd FreeRCT    # Go into the downloaded source directory.
+        $ cmake .       # Checks libraries are where they're supposed to be and replaces some strings.
+        $ make          # Let make do the heavy work.
+        $ make install  # Optional step to install the program on the system. May require root privileges.
 
 
 CMake accepts the following options:
 
-================= ======= ==========================================================================
-Name              Default Explanation
-================= ======= ==========================================================================
-OVERRIDE_VCS      OFF     Override the lack of a VCS checkout.
-ASAN              OFF     Use AddressSanitizer (https://clang.llvm.org/docs/AddressSanitizer.html).
-USERDATA_PREFIX   '.'     Directory where user data such as savegames is stored.
-================= ======= ==========================================================================
+======================= ============================= ================================================================================
+Name                    Default value                 Explanation
+======================= ============================= ================================================================================
+OVERRIDE_VCS            OFF                           Override the lack of a VCS checkout. Supported VCSs are git and bzr.
+VERSION_STRING          '0.0.0-norev'                 If OVERRIDE_VCS is enabled, use this string as the build version.
```

Comment:
Does this mean that `OVERRIDE_VCS` and `VERSION_STRING` always occur as pair? In that case, woud it make sense to combine both to a single option? (`VERSION_STRING` seems the most likely candidate to me, but maybe there are some edge cases I am missing.)

Reply:
I removed OVERRIDE_VCS and changed the priority order. If VERSION_STRING is set, this one is always used; next try to get the git revision; finally fall back to `0.0.0-revdetect-broken`.